Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/entity/deferred/request/proxy/EntityPatientDiscoveryDeferredRequestProxyWebServiceSecuredImpl.java

```diff
@@ -55,23 +56,26 @@ protected WebServiceProxyHelper createWebServiceProxyHelper() {
         return new WebServiceProxyHelper();
     }
 
+    @Override
     public MCCIIN000002UV01 processPatientDiscoveryAsyncReq(PRPAIN201305UV02 message, AssertionType assertion,
-            NhinTargetCommunitiesType targets) {
+        NhinTargetCommunitiesType targets, Properties webContextProperties) {
         LOG.debug("Begin processPatientDiscoveryAsyncReq");
         MCCIIN000002UV01 response = new MCCIIN000002UV01();
 
         try {
             String url = oProxyHelper
-                    .getUrlLocalHomeCommunity(NhincConstants.PATIENT_DISCOVERY_ENTITY_SECURED_ASYNC_REQ_SERVICE_NAME);
-            ServicePortDescriptor<EntityPatientDiscoverySecuredAsyncReqPortType> portDescriptor = new EntityPatientDiscoverySecuredAsyncReqServicePortDescriptor();
+                .getUrlLocalHomeCommunity(NhincConstants.PATIENT_DISCOVERY_ENTITY_SECURED_ASYNC_REQ_SERVICE_NAME);
+            ServicePortDescriptor<EntityPatientDiscoverySecuredAsyncReqPortType> portDescriptor
+                = new EntityPatientDiscoverySecuredAsyncReqServicePortDescriptor();
             CONNECTClient<EntityPatientDiscoverySecuredAsyncReqPortType> client = CONNECTClientFactory.getInstance()
-                    .getCONNECTClientSecured(portDescriptor, url, assertion);
-            RespondingGatewayPRPAIN201305UV02SecuredRequestType securedRequest = new RespondingGatewayPRPAIN201305UV02SecuredRequestType();
+                .getCONNECTClientSecured(portDescriptor, url, assertion);
+            RespondingGatewayPRPAIN201305UV02SecuredRequestType securedRequest
+                = new RespondingGatewayPRPAIN201305UV02SecuredRequestType();
             securedRequest.setNhinTargetCommunities(targets);
             securedRequest.setPRPAIN201305UV02(message);
 
             response = (MCCIIN000002UV01) client.invokePort(EntityPatientDiscoverySecuredAsyncReqPortType.class,
-                    "processPatientDiscoveryAsyncReq", securedRequest);
+                "processPatientDiscoveryAsyncReq", securedRequest);
         } catch (Exception ex) {
             LOG.error("Error calling processPatientDiscoveryAsyncReq: " + ex.getMessage(), ex);
```

Comment:
When you're in the code, if you see ex.getMessage() in a file you're already touching, can you change to ex.getLocalizedMessage()?  Little improvements add up over time :)